crates/semantic_index/Cargo.toml

```diff
@@ -22,6 +22,7 @@ anyhow.workspace = true
 client.workspace = true
 clock.workspace = true
 collections.workspace = true
+completion.workspace = true
```

Comment:
I'm adding the dep now in order to make sure a future PR doesn't introduce a cycle between now and then!

Reply:
You might need to add a phony reference to it in the code as well, so that `cargo-machete` doesn't complain.

Reply:
Yep, didn't realize we ran that on CI...but I'm glad we do! 😄